test/com/facebook/buck/java/intellij/ProjectIntegrationTest.java

```diff
@@ -443,19 +443,32 @@ public void testBuckProjectWithMultipleLibrariesInOneBuildFile() throws IOExcept
   }
 
   @Test
-  public void testNonexistentTarget() throws IOException {
+  public void testProjectWithColon() throws IOException {
     ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
         this,
         "project1",
         temporaryFolder);
     workspace.setUp();
 
-    expectedException.expect(HumanReadableException.class);
```

Comment:
That's good to know. Sorry about the back and forth, and thanks for pushing this through.